src/ml/neural_net/mps_compute_context.mm

```diff
@@ -88,12 +88,45 @@
 std::unique_ptr<model_backend> mps_compute_context::create_object_detector(
     int n, int c_in, int h_in, int w_in, int c_out, int h_out, int w_out,
     const float_array_map& config, const float_array_map& weights) {
-
+  float_array_map updated_config;
+  constexpr float MPS_LOSS_MULTIPLIER = 8;
```

Comment:
Let's add a comment here. Also, we should be able to define the constant in only one place, probably in `mps_graph_cnn_module.h`

Can probably just define `static constexpr float MPS_OD_LOSS_MULTIPLIER = 8.f;` somewhere